apps/studio/data/reports/v2/auth.config.ts

```diff
@@ -0,0 +1,478 @@
+        from auth_logs
+        where json_value(event_message, "$.path") = '/token'
+        group by timestamp, grant_type
+        order by timestamp desc, grant_type
+      `
+  },
+  SignUpLatency: (interval) => {
+    const granularity = analyticsIntervalToGranularity(interval)
+    return `
+        --signup-latency
+        select 
+          timestamp_trunc(timestamp, ${granularity}) as timestamp,
+          json_value(event_message, "$.auth_event.traits.provider") as provider,
+          round(avg(cast(json_value(event_message, "$.duration") as int64)) / 1000000, 2) as avg_latency_ms,
+          round(min(cast(json_value(event_message, "$.duration") as int64)) / 1000000, 2) as min_latency_ms,
+          round(max(cast(json_value(event_message, "$.duration") as int64)) / 1000000, 2) as max_latency_ms,
+          round(approx_quantiles(cast(json_value(event_message, "$.duration") as int64), 100)[offset(50)] / 1000000, 2) as p50_latency_ms,
+          round(approx_quantiles(cast(json_value(event_message, "$.duration") as int64), 100)[offset(95)] / 1000000, 2) as p95_latency_ms
+        from auth_logs
+        where json_value(event_message, "$.auth_event.action") = 'user_signedup'
+          and json_value(event_message, "$.status") = '200'
+        group by timestamp, provider
+        order by timestamp desc, provider
+      `
+  },
+  ErrorsByStatus: (interval) => {
+    const granularity = analyticsIntervalToGranularity(interval)
+    return `
+        --auth-errors-by-status
+  select 
+    timestamp_trunc(timestamp, ${granularity}) as timestamp,
+    count(*) as count,
+    response.status_code
+  from edge_logs
+    cross join unnest(metadata) as m
+    cross join unnest(m.request) as request
+    cross join unnest(m.response) as response
+  where path like '%/auth%'
+    and response.status_code >= 400 and response.status_code <= 599
+  group by timestamp, status_code
+  order by timestamp desc
+      `
+  },
+}
+
+type AuthReportFilters = {
+  status_code: NumericFilter | null
+}
+
+function filterToWhereClause(filters?: AuthReportFilters): string {
+  if (!filters) return ''
+  return ``
+}
+
+/**
+ * Transforms raw analytics data into a chart-ready format by ensuring data consistency and completeness.
+ *
+ * This function addresses several key requirements for chart rendering:
+ * 1. Fills missing timestamps with zero values to prevent gaps in charts
+ * 2. Creates a consistent data structure with `period_start` as the time axis
+ * 3. Initializes all chart attributes to 0, then populates actual values
+ * 4. Sorts timestamps chronologically for proper chart ordering
+ *
+ * @param rawData - Raw analytics data from backend queries containing timestamp and count fields
+ * @param attributes - Chart attribute configuration defining what metrics to display
+ * @returns Formatted data object with consistent time series data and chart attributes
+ */
+export function defaultAuthReportFormatter(
+  rawData: any,
```

Comment:
```suggestion
  rawData: unknown,
```

nit: I think it's better to prefer `unknown` to `any` to force ourselves to defensively code but if it's too much work to change it here then yolo 🤷🏼‍♀️